inc/saiport.h

```diff
@@ -454,7 +454,7 @@ typedef enum _sai_port_attr_t
      * @brief Media Type
      *
      * @type sai_port_media_type_t
-     * @flags CREATE_ONLY
+     * @flags CREATE_AND_SET
```

Comment:
This change was already pushed into the master branch in PR#650.  There is need to make same change in 0.9.6 for integration purposes.

CC @kcudnik / @lguohan